<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Empleados</class>
 <widget class="QMainWindow" name="Empleados">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Product Sans</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #bb1135;</string>
    </property>
    <property name="text">
     <string>Empleados</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbwEmpleados">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>630</y>
      <width>1371</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
     </font>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>ID Empleado</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nomnre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido Paterno</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido Materno</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha de nacimiento</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Puesto</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tel√©fono</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Celular</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Estado</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>590</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Empleados:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Buscar empleado:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtIdUsuario">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>151</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	color: #77767c;
	border: 2px solid  rgba(0,0,0,0.1);
	border-radius: 10px;
	padding: 10px;
}

QLineEdit:focus {
	border:2px solid #c8d6ed;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>ID Usuario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegistrar">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>160</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: #fff;
	background-color: #bb1135;
	border-radius: 5px;
	border: none;
}

QPushButton:pressed {
	background-color: #4d0000;
}</string>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtDatos">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>281</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	color: #77767c;
	border: 2px solid  rgba(0,0,0,0.1);
	border-radius: 10px;
}

QTextEdit:focus {
	border:2px solid #c8d6ed;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>500</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: #bb1135;
	border-radius: 5px;
	border: none;
}

QPushButton:pressed {
	color: #4d0000;
}</string>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Registrar empleado</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>201</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(240, 240, 240);
	border: none;
	border-bottom: 2px solid #bb1135;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Nombres</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPaterno">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>160</y>
      <width>191</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(240, 240, 240);
	border: none;
	border-bottom: 2px solid #bb1135;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Apellido paterno</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMaterno">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>160</y>
      <width>191</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(240, 240, 240);
	border: none;
	border-bottom: 2px solid #bb1135;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Apellido materno</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre(s):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>200</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apellido paterno</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>200</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apellido materno</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fecha de nacimiento:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateNacimiento">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>240</y>
      <width>141</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="displayFormat">
     <string>dd/MMM/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEmail">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>241</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(240, 240, 240);
	border: none;
	border-bottom: 2px solid #bb1135;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTelefono">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>300</y>
      <width>191</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(240, 240, 240);
	border: none;
	border-bottom: 2px solid #bb1135;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Telefono</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>310</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tel√©fono:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>380</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Puesto:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbPuesto">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>370</y>
      <width>231</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border: 1px solid #ced4da;
	border-radius: 4px;
	padding-left: 15px;
}

QComboBox::drop-down {
	border: 0px;
}

QComboBox:on {
	border: 4px solid #c2dbfe;
}

QComboBox QListView {
	font-size: 12px;
	border: 1px solid rgba(0,0,0,.1);
	padding: 5px;
	background-color: #fff;
	outline: 0px;
}

QComboBox QListView:item {
	padding-left: 10px;
	background-color: #fff;
}

QComboBox QListView:item:hover {
	background-color: #1e90ff;
}

QComboBox QListView:item:selected {
	background-color: #1e90ff;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>430</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Datos de inicio de sesi√≥n:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>510</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Usuario</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUsuario">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>470</y>
      <width>201</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(240, 240, 240);
	border: none;
	border-bottom: 2px solid #bb1135;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Nombre de usuario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtContrasenia">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>470</y>
      <width>201</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(240, 240, 240);
	border: none;
	border-bottom: 2px solid #bb1135;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Contrase√±a</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>510</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Contrase√±a</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuscar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: #bb1135;
	border-radius: 5px;
	border: none;
}

QPushButton:pressed {
	color: #4d0000;
}</string>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtRegistro">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>200</y>
      <width>281</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	color: #77767c;
	border: 2px solid  rgba(0,0,0,0.1);
	border-radius: 10px;
}

QTextEdit:focus {
	border:2px solid #c8d6ed;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar2">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>510</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: #bb1135;
	border-radius: 5px;
	border: none;
}

QPushButton:pressed {
	color: #4d0000;
}</string>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCelular">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>221</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(240, 240, 240);
	border: none;
	border-bottom: 2px solid #bb1135;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Celular</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>380</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Celular:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
